decidim-participatory_processes/app/cells/decidim/participatory_processes/process_filters_cell.rb

```diff
@@ -100,7 +100,7 @@ def explanation
         content_tag(
           :span,
           I18n.t(explanation_text, scope: "decidim.participatory_processes.participatory_processes.filters.explanations"),
```

Comment:
GH didn't leave me to add it in the next review :)

```suggestion
          I18n.t(explanation_text, scope: "decidim.participatory_processes.participatory_processes.filters.explanations")
```